app/test_main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+import pytest
+from unittest import mock
```

Comment:
check order of imports - also optimize them. No need to import mock on this line and then import from mock on line 4